advisor/src/test/kotlin/advisors/OssIndexTest.kt

```diff
@@ -123,15 +120,13 @@ class OssIndexTest : WordSpec({
 
             val result = ossIndex.retrievePackageFindings(packages).mapKeys { it.key.id }
 
-            result shouldNotBeNull {
-                keys should containExactly(COMPONENTS_REQUEST_IDS)
-
-                COMPONENTS_REQUEST_IDS.forEach { pkg ->
-                    val pkgResult = getValue(pkg)
-                    pkgResult.advisor shouldBe ossIndex.details
-                    pkgResult.vulnerabilities should beEmpty()
-                    pkgResult.summary.issues shouldHaveSize 1
-                    val issue = pkgResult.summary.issues.first()
+            result.keys shouldBe COMPONENTS_REQUEST_IDS
+            COMPONENTS_REQUEST_IDS.forEach { pkg ->
```

Comment:
Line 124 can now instead iterate on the `value` in the result, which are non-nullable, making the look-up and `shouldNotBeNull` unnecessary. 
```
result.entries.forEach { (_, advisorResult) ->
```

Reply:
While at it, consider using `forAll` instead of `forEach` for better failure output.